composer.json

```diff
@@ -50,12 +50,15 @@
         "roave/doctrine-simplecache": "^2.3",
         "slim/slim": "^3.12",
         "slim/twig-view": "^2.5",
-        "symfony/console": "^4.4"
+        "symfony/console": "^4.4",
+        "webmozart/assert": "^1.9"
     },
     "require-dev": {
-        "roave/security-advisories": "dev-master",
+        "doctrine/coding-standard": "^8.0",
+        "mikey179/vfsstream": "^1.6",
         "phpunit/phpunit": "^7.5",
```

Comment:
Why dont you update the other dependencies too? PHPunit is already in version 9, Symfony in version 5

Reply:
Thanks for the review, the changes were already coming, I just hadn't finished yet :D